### ISSUE TITLE
Edit Question: archive_v1.1.0-beta.1/general/topic-research_question/title

### ISSUE THREAD

**Issue by smohex** [FDM-UARuhr/rdmo-catalog-uaruhr#48]
**Describe the request**  
The storage colleagues expect a description of the project at this point, the main research question is not necessarily of interest at this point.

**Propose what exactly should be changed**  
Rephrase "Please briefly describe the contents of the project". Maybe the desired information is already covered by the description in the project settings? Then the question might not be needed in the archive catalog?

**Original setting or content**   
Question: Wie lautet die primäre Forschungsfrage des Projektes? / What is the main research question of the project?

**Proposed setting or content**   
New question text: Bitte beschreiben Sie kurz die Inhalte des Projekts. / Please briefly describe the contents of the project

or alternatively: delete the question in the archive catalogue

**From the following list delete everything that is NOT part of the content of your proposal. Once a subject of change is approved it will be marked accordingly in the list.**  

- [x] question text (en and ger)

**Comment by ClaraRosalie** [comment]
replace "Wie lautet die primäre Forschungsfrage des Projektes? / What is the main research question of the project?" with "Bitte beschreiben Sie kurz die Inhalte des Projekts. / Please briefly describe the contents of the project" generally in all catalogs

**Comment by smohex** [comment]
UDE/TUDO: agree to exchange the question text in all catalogs

**Comment by jmfrenzel** [comment]
To keep in line with upstream https://github.com/rdmorganiser/rdmo-catalog development we should 

1. revert he requested change in this issue and set the question content back to the original setting or content
`Wie lautet die primäre Forschungsfrage des Projektes? / What is the main research question of the project?`
2. Get the requested content of the project as being listed (per data set) in https://github.com/rdmorganiser/rdmo-catalog/blob/fc0187886bdeb00afd54f63f7b1f0caa2ea4c825/rdmorganiser/questions/rdmo.xml#L808-L827

**Comment by jmfrenzel** [comment]
@smohex would you agree that "description of the project" required by the storage colleagues is determined by the question  https://github.com/rdmorganiser/rdmo-catalog/blob/fc0187886bdeb00afd54f63f7b1f0caa2ea4c825/rdmorganiser/questions/rdmo.xml#L808-L827

**Comment by jmfrenzel** [comment]
> @smohex would you agree that "description of the project" required by the storage colleagues is determined by the question https://github.com/rdmorganiser/rdmo-catalog/blob/fc0187886bdeb00afd54f63f7b1f0caa2ea4c825/rdmorganiser/questions/rdmo.xml#L808-L827

We may also consider to retrieve the required information from multiple questions 
1.  https://github.com/rdmorganiser/rdmo-catalog/blob/fc0187886bdeb00afd54f63f7b1f0caa2ea4c825/rdmorganiser/questions/rdmo.xml#L808-L827
2. https://github.com/rdmorganiser/rdmo-catalog/blob/fc0187886bdeb00afd54f63f7b1f0caa2ea4c825/rdmorganiser/questions/rdmo.xml#L52-L63
3.  https://github.com/rdmorganiser/rdmo-catalog/blob/fc0187886bdeb00afd54f63f7b1f0caa2ea4c825/rdmorganiser/questions/rdmo.xml#L808-L827
4. ...

**Comment by smohex** [comment]
yes, I agree that https://github.com/rdmorganiser/rdmo-catalog/blob/fc0187886bdeb00afd54f63f7b1f0caa2ea4c825/rdmorganiser/questions/rdmo.xml#L808-L827 is sufficient @jmfrenzel
